src/routes/resources/resources.handlers.ts

```
@@ -609,3 +815,261 @@ export const getUsersResources: AppRouteHandler<GetUsersResources> = async (
+
+      // Queue the removal operation
+      await qstashClient.publishJSON({
+        url: `${env.APP_URL}/resource/upvote/queue`,
+        body: {
+          userId: userLoggedIn.id,
+          resourceId: id,
+          action: "remove",
+          timestamp: Date.now(),
+        },
+      });
+    } else {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(style/brace-style)